docs/PrismaMigrationTools.md

```diff
@@ -0,0 +1,49 @@
+# Prisma Migration Tools
+
+## `npm prisma:seed`
+
+### What it does
+
+This command seeds the database, which means populating the database with initial data it requires to run. It can be dummy data, basic user accounts, a default language, etc.
+
+### Why to use it
+
+While developing the application, we need some data to be present. Running this command populates the database with the data it needs.
+
+For more detailed information about [prisma seed](https://www.prisma.io/docs/guides/database/seed-database), click the link.
+
+## `npm prisma:reset`
+
+### What it does
+
+This command drops the database if possible or performs a soft reset if the database cannot be deleted. If the database is dropped, it creates a new database with the same name and runs all migrations and seed scripts.
```

Comment:
Say a little more about how because it runs seeding, then if this is successful you don't need to run the seed command again. Also mention that you use reset if you've sufficiently modified the data in the front end and want to go back to a clean slate with data populated.

Reply:
Oops maybe not all in that line, but in general